### PR TITLE
#85 Initial version of Glossary Author explorer

### PR DESCRIPTION
**Pull request by davidradl**

### CODE REVIEW

File: docs/design/glossary-author/README.md [hunk 1]
```diff
@@ -3,14 +3,69 @@
...
+
+
+
+
+
+
+
+
+
+
```
Comment (sarbull) [inline docs/design/glossary-author/README.md:68]: 👍 we need more of this kind of documentation

Comment (davidradl) [reply docs/design/glossary-author/README.md]: Sorry I mistakenly added a confused icon. Sorry for the confusion @sarbull

**Comment by davidradl** [comment]
@planetf1  @sarbull I will merge at the end of tomorrow if I hear no more feedback.

**Review by planetf1 — APPROVED** [review]
No particular feedback from me - not had a chance and not JS developer. Since it's a new function I think it's fine to merge & then critique/review ?

**Comment by davidradl** [comment]
@planetf1 @sarbull OK I will merge.

**Pull request merged**